x/exchange/keeper/orders_test.go

```diff
@@ -1716,8 +1716,8 @@ func (s *TestSuite) TestKeeper_SetOrderExternalID() {
 			marketID:      1,
 			orderID:       1,
 			newExternalID: strings.Repeat("I", exchange.MaxExternalIDLength+1),
-			expErr: fmt.Sprintf("invalid external id %q: max length %d",
-				strings.Repeat("I", exchange.MaxExternalIDLength+1), exchange.MaxExternalIDLength),
+			expErr: fmt.Sprintf("invalid external id %q (length %d): max length %d",
+				"IIIII...IIIII", exchange.MaxExternalIDLength+1, exchange.MaxExternalIDLength),
```

Comment:
I considered that when writing `SetOrderExternalID` (and `TestKeeper_SetOrderExternalID`). I felt the concern was negligible, though, because an `external_id` needs to at least be able to hold a UUID. Since a UUID 36 characters, `exchange.MaxExternalIDLength` should never be less than `36`. That hard-coded value is the first 5 and last 5 characters of the long `external_id` (with `...` between them). So changing `MaxExternalIDLength` will only cause problems if it's changed to something less than `10`, which should never happen anyway.